src/main/addons/autoav.ts

```diff
@@ -138,10 +190,12 @@ export default class AutoAV {
 
     // Log a message
     private log(msg: string, severity: 'out' | 'err' = 'out') {
-        fs.appendFileSync(path.join(appdataPath, 'logs', `autoav.${severity}.log`), `${new Date().toISOString()} - ${msg}\n`, { flag: 'a' });
+        if (!this.logs) throw new Error('Loggers have not been configured.');
```

Comment:
@Techno11 just a heads up that addons are now logging via `electron-log`. `err` and `out` are still in separate files for each addon, and archival still works the same way, but format inside the file will be slightly different